lib/web_ui/lib/initialization.dart

```diff
@@ -130,7 +130,8 @@ void webOnlySetPluginHandler(Future<void> Function(String, ByteData?, PlatformMe
 }
 
 /// A function which takes a unique `id` and creates an HTML element.
-typedef PlatformViewFactory = html.Element Function(int viewId);
+/// !!!! before landing ensure this won't break flutter.
```

Comment:
Hmmm.... what's the plan here? We might need a broader discussion on plugins. I suspect this might break stuff. @yjbanov might be more privy to the subtleties here

Reply:
@yjbanov and I spoke offline, we don't think this is really used, and if we break anyone there is a very clear work around(just recreate the type def locally), so we agreed to try just removing it.